parser/SqlParser.ypp

```diff
@@ -239,6 +240,7 @@ void NotSupported(const YYLTYPE *location, yyscan_t yyscanner, const std::string
 %token TOKEN_FROM;
 %token TOKEN_FULL;
 %token TOKEN_GROUP;
+%token TOKEN_GB;
```

Comment:
Is this TOKEN_GB still being used in the Parser grammar? If not, then maybe we can remove it.